src/Rules/Operator/NamedArgumentOperatorRule.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\Rules\Operator;
+
+use Composer\InstalledVersions;
+use Composer\Semver\VersionParser;
+use TwigCsFixer\Rules\AbstractFixableRule;
+use TwigCsFixer\Token\Token;
+use TwigCsFixer\Token\Tokens;
+
+/**
+ * Ensures named arguments use `:` syntax instead of `=` (For `twig/twig >= 3.12.0`).
+ */
+final class NamedArgumentOperatorRule extends AbstractFixableRule
+{
+    public function __construct()
+    {
+        if (!InstalledVersions::satisfies(new VersionParser(), 'twig/twig', '>=3.12.0')) {
+            throw new \InvalidArgumentException('Named argument with semi colons requires twig/twig >= 3.12.0');
+        }
```

Comment:
_**If**_ you want to, you can use the Twig Environment constants for this.

Reply:
Not sure it can be trusted, when we see mistake like https://github.com/twigphp/Twig/blob/v3.11.0/src/Environment.php#L48